src/components/HowItWorks/index.js

```diff
@@ -0,0 +1,73 @@
+import {
+  Button,
+  Grid,
+  Hidden,
+  Typography,
+  useMediaQuery,
+} from "@material-ui/core";
+import { useTheme } from "@material-ui/core/styles";
+import Image from "next/image";
+import PropTypes from "prop-types";
+import React from "react";
+
+import visualsImg from "@/pesayetu/assets/images/Component1236.svg";
+import howItWorksBg from "@/pesayetu/assets/images/sat-mtKenya@2x.png";
+import Player from "@/pesayetu/components/HowItWorks/Player";
+import useStyles from "@/pesayetu/components/HowItWorks/useStyles";
+import Section from "@/pesayetu/components/Section";
+
+function HowItWorks({ title, ctaText, description, href, ...props }) {
+  const classes = useStyles(props);
+
+  const theme = useTheme();
+  const isDownSM = useMediaQuery(theme.breakpoints.down("sm"));
+
+  return (
+    <div className={classes.root}>
+      <div className={classes.background}>
+        <Image src={howItWorksBg} layout="fill" />
+      </div>
+      <Hidden lgUp smDown implementation="css">
+        <div className={classes.tabletWhite} />
+      </Hidden>
+      <Section classes={{ root: classes.section }}>
+        <Grid container direction={isDownSM ? "column-reverse" : "row"}>
+          <Grid item xs={12} md={7} lg={6} className={classes.content}>
+            <Player {...props} />
+            <Typography className={classes.title} variant="h4">
+              {title}
+            </Typography>
+            <Typography variant="body2" className={classes.description}>
+              {description}
+            </Typography>
+            <Button href={href} variant="text">
+              {ctaText}
+            </Button>
+          </Grid>
+          <Grid item lg={1} />
```

Comment:
Won't `justify="space-between"` achieve the same thing?

Reply:
@kilemensi it could on desktop, but then I will need different justification on tablet & mobile. So this is simpler for this case